alacritty/src/main.rs

```diff
@@ -210,17 +211,15 @@ fn alacritty(mut options: Options) -> Result<(), Box<dyn Error>> {
     // order.
     //
     // FIXME: Change PTY API to enforce the correct drop order with the typesystem.
-    drop(processor);
-
-    // FIXME patch notify library to have a shutdown method.
-    // config_reloader.join().ok();
-
     // Without explicitly detaching the console cmd won't redraw it's prompt.
     #[cfg(windows)]
     unsafe {
         FreeConsole();
     }
 
+    // FIXME patch notify library to have a shutdown method.
+    // config_reloader.join().ok();
```

Comment:
I can't believe this comment is still around. :smile: 

Config reloader isn't even a valid identifier for any variables in this scope.

It is definitely possible to shut down the watcher cleanly at this point and I might throw up a patch for that myself in a second. For this PR I recommend just removing this entirely.